where-to-travel/src/main/webapp/script.js

```diff
@@ -447,46 +502,72 @@ function populatePlaces(placeArray) {
       focusedCard.scrollIntoView({behavior: 'smooth', block: 'center'});
     });
 
-    placeMarker.addListener('mouseover', function () {
-      placeMarker.setIcon(SELECTED_PIN_PATH);
-    });
+      placeMarker.addListener('mouseover', function () {
```

Comment:
Undo these indentation changes? I believe they were originally aligned.

Reply:
fixed in d6b47d2

Reply:
no it isnt, i made a mistake

Reply:
fixed in c5463ef